src/components/multiselect.vue

```diff
@@ -365,8 +367,7 @@ const $dropdown = computed(() => $(dropdown.value));
 onMounted(() => {
   $dropdown.value.on('shown.bs.dropdown', syncWithModelValue);
   $dropdown.value.on('hidden.bs.dropdown', () => {
```

Comment:
Currently, it doesn't look like the `changes` set is cleared in this case. Before, it wasn't possible to hide the dropdown without emitting changes, so it was the sole responsibility of `emitIfChanged` to call `changes.clear()`. But now we need to clear `changes` in this case as well.

One question: should `changes.clear()` be called here or in the `syncWithModelValue()` function? Maybe it'd be better to call it here so that the values in `changes` aren't held onto unnecessarily.